ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/block/BlockManager.java

```diff
@@ -168,6 +170,7 @@ private void importBlockIgnoringResult(final SignedBeaconBlock block) {
     if (pendingBlocks.contains(block)
         || futureBlocks.contains(block)
         || recentChainData.containsBlock(block.getRoot())) {
+      // TODO: Need to make sure the result sets optimistic flag correctly
```

Comment:
is this something we should have a ticket reference for? it'll never get done potentially if its just here...

Reply:
I meant to go back and fix it... probably better to do as a follow up though so yeah will log a ticket and remove the TODO.